tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -198,15 +198,15 @@ Feature: Sharing files and folders with internal groups
     And as "Brian" file "/Shares/lorem.txt" should not exist in the server
     But as "Carol" file "/Shares/lorem.txt" should exist in the server
 
-  @issue-ocis-1317
+  @issue-ocis-1317 @skipOnOCIS
   Scenario: Auto-completion for a group that is excluded from receiving shares
     Given group "system-group" has been created in the server
     And user "Alice" has created folder "simple-folder" in the server
+#   exclude-groups-from-sharing configuration is not available now in the oCIS webUI
```

Comment:
I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue